assets/src/blocks/Submenu/SubmenuEditor.js

```diff
@@ -69,11 +115,24 @@ const renderEdit = (attributes, setAttributes) => {
   );
 }
 
-const renderView = (attributes, setAttributes) => {
+// We can put the other blocks that can have a header inside in here along with the attribute containing the heading text.
+// Then we can also filter those to include them in the menu.
+const blockTypesWithHeadings = [
```

Comment:
I included this logic to include headers in our blocks, to achieve the same behavior as the previous block. But we could check existing content to see if there actually are pages that have such titles in the submenu. If there aren't we could consider dropping this part, as it could be confusing to the user. They don't see the heading level in the editor for these headings like they do in heading blocks.

If we keep it then we need to add the other blocks with headings here too.